addons/pinniped/config-controller/controllers/v1_controller.go

```diff
@@ -135,9 +136,11 @@ func (c *PinnipedV1Controller) reconcileAddonSecret(ctx context.Context, cluster
 	// also check if cluster is scheduled for deletion, if so, delete addon secret on mgmt cluster
 	if pinnipedInfoCM.Data == nil || !cluster.GetDeletionTimestamp().IsZero() {
 		log.V(1).Info("deleting secret")
-		if err := c.reconcileDelete(ctx, client.ObjectKeyFromObject(cluster), log); err != nil {
+		if err := c.reconcileDelete(ctx, secretNameFromClusterName(client.ObjectKeyFromObject(cluster)), log); err != nil {
+			log.Info("failed to delete, error")
```

Comment:
We actually already log the error in `reconcileDelete` before returning the error here.  There doesn't seem to be a need to log it again since we aren't adding any new information.

Reply:
cool. ill remove.